services/QuillLMS/app/controllers/api/v1/session_feedback_histories_controller.rb

```diff
@@ -1,8 +1,7 @@
 class Api::V1::SessionFeedbackHistoriesController < Api::ApiController
-  # GET /feedback_histories.json?page=1&activity_id=33
+  # GET /feedback_histories.json?page=1&activity_id=33&start_date=2021-04-18T03:00:00.000Z&end_date=2021-05-18T03:00:00.000Z
```

Comment:
Do we have this API documented in Notion anywhere?  If so, can we get this update included there, too?

Reply:
updated!